File: py/Vehicle.py
class Vehicle:
    def __init__(self, plat_nomor, merk, tahun_produksi, warna):
        self.plat_nomor = plat_nomor
        self.merk = merk
        self.tahun_produksi = tahun_produksi
        self.warna = warna

    def __str__(self):
        return f"""
Plat Nomor: {self.plat_nomor}
Merk: {self.merk}
Tahun Produksi: {self.tahun_produksi}
Warna: {self.warna}
"""

    def to_table_row(self):
        return f"| {self.plat_nomor:<10} | {self.merk:<14} | {self.tahun_produksi:<5} | {self.warna:<5} |"

File: cpp/vehicle.cpp
#pragma once

#include <iostream>
#include <vector>
#include <iomanip>

using namespace std;

class Vehicle
{
    public:
        string plat_nomor;
        string merk;
        string tahun_produksi;
        string warna;

        Vehicle()
        {
        }

        Vehicle(string plat_nomor, string merk, string tahun_produksi, string warna) : 
            plat_nomor(plat_nomor), merk(merk), tahun_produksi(tahun_produksi), warna(warna)
        {
        }

        string toString()
        {
            return "Plat Nomor: " + plat_nomor + "\nMerk: " + merk + "\nTahun Produksi: " + tahun_produksi + "\nWarna: " + warna + "\n";
        }

        string toTableRow()
        {
            return "| " + plat_nomor + "  | " + merk + "  | " + tahun_produksi + "  | " + warna + " |";
        }
};